cmd/ocm/create/cluster/cmd.go

```diff
@@ -1071,22 +1071,47 @@ func promptExistingGCPVPC(fs *pflag.FlagSet, connection *sdk.Connection) error {
 		}
 	}
 
-	if args.existingVPC.Enabled || wasGCPNetworkReceived() {
-		err = arguments.PromptString(fs, "vpc-name")
-		if err != nil {
-			return err
-		}
+	if !args.existingVPC.Enabled && !wasGCPNetworkReceived() {
```

Comment:
What happens if interactive is enabled but the top level EnableExistingVPC is not enabled.

Will it be caught somewhere else and prompt user for enabling existing VPC

Reply:
nm, I see the interactive prompt for the EnableExistingVPC right   above